cdptools/databases/cloud_firestore_database.py

```diff
@@ -77,20 +77,24 @@ def __init__(
         # With credentials:
         if credentials_path:
             self._initialize_creds_db(credentials_path, name)
+
+            # We fetch all tables in case some tables exist in target, but not in source
+            # Path returns a tuple, and root collection path is in first index
+            self._tables = [coll._path[0] for coll in self._root.collections()]
         elif project_id:
             self._credentials_path = None
             self._project_id = project_id
             self._db_uri = FIRESTORE_BASE_URI.format(project_id=project_id)
         else:
             raise exceptions.MissingParameterError(["project_id", "credentials_path"])
 
-        self._tables = [
+        self._cdp_tables = [
```

Comment:
I would move this to a global variable in the `database.py` file so they can be imported into any database sub class. And also move this into the `else` condition so that it is the default when not using credentials.
```suggestion
        self._tables = CDP_TABLES
```

Reply:
Gotcha yeah that's a better design pattern